src/pages/products/sgid/elevation/lidar.astro

```diff
@@ -653,6 +674,22 @@ const projectAssets = {
       For more information about Lidar visit <a href="/documentation/sgid/exploring-lidar/">Exploring Lidar</a>.
     </p>
   </Section>
+  <Section {...page.section[28]}>
+    <a href={westernEastern2022Map.src} slot="image">
+      <Image src={westernEastern2022Map} width="400" alt="A map with project boundary" class="w-full max-w-[400px]" /></a
+    >
+    <p>
+      During 2022 UGRC, the Utah Division of Emergency Management, the Natural Resource Conservation Service (NRCS), the
+      U.S. Forest Service (USFS), and the Federal Emergency Management Agency (FEMA) acquired ~19,466 square miles of 2
+      and 8 points per meter Quality Level 1 & 2 Lidar of portions Utah. The 0.5 and 1 meter resolution bare earth DEMs
```

Comment:
**thought**: the `of portions Utah.` part of the sentence reads funny. Does it need another of? 

> yada yada acquired Lidar of portions of Utah?

Reply:
good catch and I see some other changes needed